src/Files.App/Actions/Open/OpenClassicPropertiesAction.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.Runtime.InteropServices;
+using static Files.App.Helpers.Win32PInvoke;
+
+namespace Files.App.Actions
+{
+	internal sealed class OpenClassicPropertiesAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		public string Label
+			=> "OpenClassicProperties".GetLocalizedResource();
+
+		public string Description
+			=> "OpenClassicPropertiesDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph
+			=> new(themedIconStyle: "App.ThemedIcons.Properties");
+
+		public HotKey HotKey
+			=> new(Keys.Enter, KeyModifiers.AltShift);
+
+		public bool IsExecutable =>
+			context.PageType is not ContentPageTypes.Home &&
+			!(context.PageType is ContentPageTypes.SearchResults && 
+			!context.HasSelection);
+
+		public OpenClassicPropertiesAction()
+		{
+			context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+			context.PropertyChanged += Context_PropertyChanged;
+		}
+
+		public Task ExecuteAsync(object? parameter = null)
+		{
+			if (context.HasSelection && context.SelectedItems is not null)
+			{
+				foreach (var item in context.SelectedItems)
+				{
+					SHELLEXECUTEINFO info = new SHELLEXECUTEINFO();
```

Comment:
I'd recommend using default instead of new.

Reply:
What do you mean?

Reply:
SHELLEXECUTEINFO shellExecuteInfo = default;
```suggestion
					SHELLEXECUTEINFO info = default;
```